feature-libs/product/_index.scss

```diff
@@ -0,0 +1,14 @@
+// we require a few bootstrap files for the CSS in this code
+@import '~bootstrap/scss/functions';
+@import '~bootstrap/scss/variables';
+@import '~bootstrap/scss/_mixins';
+
+@mixin visible-focus {
```

Comment:
Do we need this? And whole bootstrap dependency?

Reply:
No, removed